### PR TITLE
Document fake delegates

### LINKED ISSUE #1320: Document that delegates can be faked
As @zvirja [points out](https://github.com/AutoFixture/AutoFixture/issues/984#issuecomment-366633722), we [don't document](http://fakeiteasy.readthedocs.io/en/stable/what-can-be-faked/) that FakeItEasy can fake delegates.

### PR DESCRIPTION
**Pull request by thomaslevesque**
Fixes #1320

### CODE REVIEW

File: docs/what-can-be-faked.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ following entities can be faked:
     * are not sealed,
     * are not static, and
     * have at least one public or protected constructor whose arguments FakeItEasy can construct or obtain
-
+* delegates
```
Comment (blairconrad) [inline docs/what-can-be-faked.md]: nit: indentation

Comment (blairconrad) [reply docs/what-can-be-faked.md]: (no need for a fixup; amend away)

Comment (blairconrad) [inline docs/what-can-be-faked.md]: I'd made a comment about bad indentation. My mistake. My context was too narrow. This is great.

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque!

**Comment by thomaslevesque** [comment]
Thanks for the merge!

**Pull request merged**